torch/_inductor/config.py

```diff
@@ -946,6 +946,9 @@ class aot_inductor:
 
     package: bool = False
 
+    # Dictionary of metadata users might want to save to pass to the runtime.
+    metadata: Dict[str, str] = {}
```

Comment:
We are putting more and more "non-config" global data here. OK for this PR, but we should do some refactoring at some point.